docs/deployment-process.md

```diff
@@ -121,6 +121,24 @@ cloud.gov UAA application for its users.
 Tock uses the cloud.gov service account service to provide deployer accounts for
 staging and production environments.
 
+### Code review
+
+Submissions to the Tock codebase are made via GitHub, and are only accepted into the main
+branch after review. At least one approving review is required before a branch is merged
+to main, and this restriction is enforced by Tock's GitHub settings.
+
+Code review covers both code (such as Python or JavaScript) and configuration (such as
+Dockerfiles, CloudFoundry manifest files, etc.).
+
+Code review includes assessment of:
```

Comment:
```suggestion
Code reviews should be conducted [following the 18F Engineering Guide](https://engineering.18f.gov/code-review/) and include an assessment of:
```

Reply:
Just cause I really like how the Eng guide spells out how to conduct good code reviews. Feel free to rephrase or reject.

Reply:
Good suggestion!